index.html

```diff
@@ -6,15 +6,128 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <link rel="stylesheet" type="text/css" href="css/grid.css">
-    <!-- Add a link to your custom CSS here -->
+    <link rel="stylesheet" type="text/css" href="css/main.css">
+    <link rel="stylesheet" type="text/css" href="https://stackpath.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css">
+        <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
 </head>
 
 <body>
+    
+<main>
+    
+    <header>       
+        <div class="container">
+            <div class="row">  
+                <div class="col-3">
+                    <h3>ALLAN JOHNSON</h3>
+                    <p>Personal Life Coach</p>
+                </div>
+                    <div class="col-6">
+                     <nav class="navigation-bar">
+                        <ul>
+                         <li><a href="#">MENU</a>
```

Comment:
Do you want the menu to be a link to another page? As the other bits in the list are hyperlinked to certain pages it is unlikely the menu needs it too